### PR TITLE
Vision Product Search

### PR DESCRIPTION
**Pull request by nirupa-kumar**
Tutorial added

### CODE REVIEW

**Comment by nirupa-kumar** [comment]
@nnegrey Please approve.

File: vision/product-search/cloud-client/resources/product_sets.csv [hunk 1]
```diff
@@ -0,0 +1,2 @@
+"gs://java-docs-samples-testing/product-search/shoes_1.jpg","fake_product_set_id_for_testing","fake_product_id_for_testing_1","apparel","style=womens","0.1,0.1,0.9,0.1,0.9,0.9,0.1,0.9"
+"gs://java-docs-samples-testing/product-search/shoes_2.jpg","fake_product_set_id_for_testing","fake_product_id_for_testing_2","apparel",,
```
Comment (nnegrey) [inline vision/product-search/cloud-client/resources/product_sets.csv]: Double comma at the end of this line?

File: vision/product-search/cloud-client/resources/indexed_product_sets.csv [hunk 2]
```diff
@@ -0,0 +1,2 @@
+"gs://java-docs-samples-testing/product-search/shoes_1.jpg","indexed_product_set_id_for_testing","indexed_product_id_for_testing_1","apparel","style=womens","0.1,0.1,0.9,0.1,0.9,0.9,0.1,0.9"
+"gs://java-docs-samples-testing/product-search/shoes_2.jpg","indexed_product_set_id_for_testing","indexed_product_id_for_testing_2","apparel",,
```
Comment (nnegrey) [inline vision/product-search/cloud-client/resources/indexed_product_sets.csv]: Double comma at the end of this line?

File: vision/product-search/cloud-client/src/main/java/com/example/vision/ProductInProductSetManagement.java [hunk 3]
```diff
@@ -0,0 +1,241 @@
...
+  // [START product_search_list_products_in_product_set]
+  /**
+   * List all products in a product set.
+   *
+   * @param projectId - Id of the project.
+   * @param computeRegion - Region name.
+   * @param productSetId - Id of the product set.
+   * @throws IOException - on I/O errors.
+   */
+  public static void listProductsInProductSet1(
```
Comment (nnegrey) [inline vision/product-search/cloud-client/src/main/java/com/example/vision/ProductInProductSetManagement.java]: Remove the 1

**Pull request merged**